setup.py

```diff
@@ -137,6 +137,27 @@ def reduce_requirements(key):
 # Get __version__ variable
 exec(open(os.path.join(here, 'cupy', '_version.py')).read())
 
+CLASSIFIERS = """\
+Development Status :: 5 - Production/Stable
+Intended Audience :: Science/Research
+Intended Audience :: Developers
+License :: OSI Approved :: MIT License
+Programming Language :: Python
+Programming Language :: Python :: 2
+Programming Language :: Python :: 2.7
```

Comment:
Shouldn't you add these two lines only when backporting to v6? In master/v7 the support for PY2 is dropped.

Reply:
Thanks, I fixed.

Reply:
Thanks, I'll add it in the backport.